build.gradle

```diff
@@ -217,6 +217,9 @@ dependencies {
         implementation 'bio.terra:terra-common-lib:0.0.16-SNAPSHOT'
     }
 
+    implementation group: "bio.terra", name: "terra-resource-buffer-client", version: "0.4.3-SNAPSHOT"
+    implementation "org.glassfish.jersey.inject:jersey-hk2:2.30.1"
```

Comment:
We should standardize if we use the splitted `group`, `name`, `client` declaration or the all-in-one declaration. I prefer the split one because its a wee bit more readable and easier to include variables in (ex: for versions)